R/git-utils.R

```diff
@@ -23,6 +23,20 @@ uses_git <- function(path = proj_get()) {
   !is.null(git2r::discover_repository(path))
 }
 
+# Remotes ------------------------------------------------------------------
+git_remotes <- function(path = proj_get()) {
```

Comment:
These definitely shouldn't take a path, but should instead use `git_repo()`

Reply:
OK done, here and many other places.